src/test/java/teammates/test/pageobjects/InstructorFeedbacksPage.java

```diff
@@ -411,16 +408,9 @@ public String getResponseValue(String courseId, String sessionName) {
     
     public void verifyResponseValue(String responseRate, String courseId, String sessionName) {
         int sessionRowId = getFeedbackSessionRowId(courseId, sessionName);
-        WebDriverWait wait = new WebDriverWait(browser.driver, 10);
-        try {
-            wait.until(ExpectedConditions.textToBePresentInElement(
-                    browser.driver.findElement(
-                            By.xpath("//tbody/tr[" + (int) (sessionRowId + 1)
-                            + "]/td[contains(@class,'session-response-for-test')]")),
-                            responseRate));
-        } catch (TimeoutException e) {
-            fail("Not expected message");
-        }
+        waitForTextContainedInElementPresence(
+                By.xpath("//tbody/tr[" + (sessionRowId + 1) + "]/td[contains(@class,'session-response-for-test')]"),
```

Comment:
![screen shot 2016-06-06 at 11 52 23 am](https://cloud.githubusercontent.com/assets/7261051/15811610/3e2a020e-2bdd-11e6-95e3-09d7eca06b35.png)

I agree that css is clearer, but I'm causing inconsistency if I do that here. We can fix this any other time.